README.md

```diff
@@ -1,2 +1,45 @@
+![GitHub issues](https://img.shields.io/github/issues-raw/softonic/homing-pigeon-chart)
+
+### Overview
+
+| Parameter                                           | Description                                                   | Default                            |
+| --------------------------------------------------- | ------------------------------------------------------------- | ---------------------------------- |
+| `image.repository`                                  | Image repository                                              | `softonic/homing-pigeon`           |
+| `image.tag`                                         | Tag used for the image                                        | `0.1.0`                            |
+| `image.pullPolicy`                                  | Image pull policy                                             | `IfNotPresent`                     |
+| `resources`                                         | Resources policy                                              | `{}`                               |
+| `nodeSelector`                                      | Node Selector for homing pigeon deployment                    | `{}`                               | 
+| `tolerations`                                       | Tolerations for homing pigeon deployment                      | `[]`                               | 
+| `affinity`                                          | Affinity for homing pigeon deployment                         | `{}`                               | 
+| `nameOverride`                                      | Override name of chart                                        | `""`                               |
+| `fullnameOverride`                                  | Override full name of chart                                   | `""`                               | 
+| `replicaCount`                                      | Number of replicas for homing pigeon deployment               | `1`                                | 
+| `messageBufferLen`                                  | Buffer length for write buffer                                | `400`                              | 
+| `ackBufferLen`                                      | Buffer length for acks                                        | `200`                              | 
+| `reader.rabbitmq.enabled`                           | Enable RabbitMQ reader                                        | `false`                            | 
+| `reader.rabbitmq.consumerName`                      | Consumer identifier label                                     | `""`                               | 
+| `reader.rabbitmq.url`                               | RabbitMQ url                                                  | `""`                               | 
+| `reader.rabbitmq.deadLettersExchangeName`           | RabbitMQ reader dead letters exchange name                    | `""`                               | 
```

Comment:
Maybe someth long more useful like dead letter for nack messages would be good.

Reply:
Dead letter are only nacked messages by definition, however the thing that is not specified is that *all* nacked messages will be dead letters (instead of retried ones for example).

What do you suggest?

Reply:
ACtually deadletters is not just for nack messages. In this configuration is just for nack messages. So I would describe the purpose of this exchange.